docs/developer-guide/views-examples.md

```diff
@@ -0,0 +1,164 @@
+---
+description: Learn about the most used custom view examples of Lite XL.
+---
+
+## Example: ToolbarView
+
+brief introduction
```

Comment:
Surely this is a placeholder for something right? You could describe what the plugin does.